### PR TITLE
CONSOLE-3090: [Dark Theme]  This is to reassign the correct blue and orange heading colors that are used in the getting started sections on both Admin and Dev console

### PR DESCRIPTION
**Pull request by sg00dwin**
These were previously updated in https://github.com/openshift/console/pull/11151, but inadvertently assigned to darker shade css properties. Note the purple vars are correct.
This only effects the light theme colors. Dark theme are assigned the correct color variables.

**Before**
adm-gs-light-before
dev-gs-light-before

**After**
adm-gs-light
dev-gs-light

**Dark**
adm-gs-dark
dev-gs-dark

### CODE REVIEW

**Comment by sg00dwin** [comment]
/label docs-approved
/label px-approved

We have docs- and px-approval for the epic.

**Comment by sg00dwin** [comment]
QE Approver:
/assign @yanpzhan

**Review by rhamilto — APPROVED** [review]
/lgtm

**Pull request merged**
